packages/twig-new/package.json

```diff
@@ -1,14 +1,57 @@
 {
-  "name": "twig-new",
-  "private": true,
-  "version": "0.0.0",
+  "name": "@ilo-org/twig-new",
+  "version": "1.2.0",
   "type": "module",
+  "license": "Apache-2.0",
+  "description": "Twig components for the ILO's Design System",
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://twig.ui.ilo.org",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/international-labour-organization/designsystem.git",
+    "directory": "packages/twig"
+  },
+  "keywords": [
+    "ui_patterns",
+    "prototyping",
+    "twig",
+    "designsystem",
+    "components"
+  ],
+  "contributors": [
+    {
+      "name": "Justin Smith",
+      "email": "smithj@ilo.org",
+      "url": "https://github.com/justintemps"
+    },
+    {
+      "name": "Prabashwara Seneviratne",
+      "url": "https://www.bash.lk"
+    },
+    {
+      "name": "Shashika Boteju",
+      "email": "Shashikaboteju3@gmail.com",
+      "url": "https://github.com/Shashika6"
+    },
+    {
+      "name": "Giorgi Kapanadze",
+      "email": "doublegkapanadze@gmail.com",
+      "url": "https://github.com/ggkapanadze"
+    }
+  ],
   "scripts": {
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build"
+    "build-storybook": "storybook build",
```

Comment:
I think we need to change the output directory to support the current deployment workflow, the old one was outputted in `/storybook-static` 

<img width="771" alt="Google Chrome 2024-07-25 13 40 34" src="https://github.com/user-attachments/assets/10d133c1-bc96-48cc-b7ff-7a78942973c5">
https://storybook.js.org/docs/api/cli-options